Source/WebCore/en.lproj/Localizable.strings

```diff
@@ -88,6 +88,9 @@
 /* Label to describe the number of files selected in a file upload control that allows multiple files */
 "%d files" = "%d files";
 
+/* Label for PDF page number indicator. */
+"%d of %d" = "%d of %d";
```

Comment:
This probably won't  be localizable in some languages.

Reply:
That's possible, yes, I'm not fully sure how to evaluate this. I just lifted our existing label text.

Reply:
Fixed by adopting positional arguments!